homeassistant/components/backup/store.py

```diff
@@ -45,4 +46,4 @@ def save(self) -> None:
     @callback
     def _data_to_save(self) -> StoredBackupData:
         """Return data to save."""
-        return {"config": self._manager.config.data.to_dict()}
+        return {"backups": {}, "config": self._manager.config.data.to_dict()}
```

Comment:
Shouldn't we store the known backups from the manager?

Reply:
👍